docs/fastdds/dds_layer/core/status/status.rst

```diff
@@ -3,3 +3,630 @@
+| total_count              | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| total_count_change       | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| current_count            | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| current_count_change     | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| last_publication_handle  | ``InstanceHandle_t``                    |
++--------------------------+-----------------------------------------+
+
+* **total_count**: Total cumulative count of remote :ref:`DataWriters<dds_layer_publisher_dataWriter>`
+  that have been discovered publishing on the same :ref:`dds_layer_topic_topic` and has
+  a common partition and a compatible QoS.
+  They may not all be matched at the moment.
+
+* **total_count_change**: The change in **total_count** since
+  the last time |on_subscription_matched| was called or the status was read.
+  It can only have zero or positive values.
+
+* **current_count**: The number of remote :ref:`DataWriters<dds_layer_publisher_dataWriter>`
+  currently matched to the :ref:`dds_layer_subscriber_dataReader`.
+
+* **current_count_change**: The change in **current_count** since
+  the last time |on_subscription_matched| was called or the status was read.
+  It can have positive or negative values.
+
+* **last_publication_handle**: Handle to the last :ref:`dds_layer_publisher_dataWriter`
+  that matched the :ref:`dds_layer_subscriber_dataReader`.
+  If no matching ever happened, it will have value ``c_InstanceHandle_Unknown``.
+
+
+.. _dds_layer_core_status_qosPolicyCount:
+
+QosPolicyCount
+^^^^^^^^^^^^^^
+
+This structure holds a counter for a policy.
+It is used to keep track of the number of times a remote policy was found to be incompatible with
+the local one.
+
+List of data members:
+
++--------------------------+-----------------------------------------+
+| Data Member Name         | Type                                    |
++==========================+=========================================+
+| policy_id                | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| count                    | ``int32_t``                             |
++--------------------------+-----------------------------------------+
+| last_policy_id           | ``uint32_t``                            |
++--------------------------+-----------------------------------------+
+| policies                 | ``std::vector<QosPolicyCount>``         |
++--------------------------+-----------------------------------------+
+
+* **policy_id**: The ID of the policy.
+
+* **count**: The counter value for the policy.
```

Comment:
Is this out of place?

Reply:
Out of place and with text that didn't correspond to this class :anguished: